### PR TITLE
Expose storage account in metric

### PR DESCRIPTION
**Pull request by achaikou**
Playground grafana is updated with respective panel, production grafana will be changed after merge (might need to discuss the need to exclude test storage account from all the metrics)

### CODE REVIEW

File: internal/metrics/metrics.go [hunk 1]
```diff
@@ -82,11 +84,35 @@ func NewGinMiddleware(metrics *Metrics) gin.HandlerFunc {
...
+	for _, vdsURL := range vdsURLs {
+		parsedURL, err := url.Parse(vdsURL)
+		if err != nil {
+			storageAccounts = append(storageAccounts, vdsURL)
+			continue
+		}
+		storageAccounts = append(storageAccounts, parsedURL.Hostname())
+	}
+
+	slices.Sort(storageAccounts)
```
Comment (yngve793) [inline internal/metrics/metrics.go:103]: Why are these sorted? Is the user expect these to be returned in alphabetical order?

**Review by yngve793 — COMMENTED** [review]
Looks good. 
Are these storages accounts only used for monitoring. 
Can there be duplicates and if so is it desired to have multiple instances of same storage account.

File: internal/metrics/metrics.go (hunk 1 above)
Comment (achaikou) [reply internal/metrics/metrics.go:103]: Because apparently by sorting and using compact one can remove duplicates using slice package.

**Review by achaikou — COMMENTED** [review]
This metric is exposed for monitoring only.

Not sure about another question, but I decided to remove duplicates from the requests (double vds source case) as we are interested in how many requests go to our storage accounts and how many users use their own.
Maybe actually should have counted it 2 times, but decided that it is more likely that both vds would go to the same storage account, so lets count it once not twice.

File: internal/metrics/metrics.go (hunk 1 above)
Comment (yngve793) [reply internal/metrics/metrics.go:103]: Ok, fine.

**Review by yngve793 — APPROVED** [review]
Tests passes and code looks good.

**Pull request merged**